doc/api/esm.md

```diff
@@ -1303,7 +1303,7 @@ the CommonJS loader. One of the behavior differences is automatic resolution
 of file extensions and the ability to import directories that have an index
 file.
 
-The `--experimental-specifier-resolution=[mode]` flag can be used to customize
+The `--experimental-specifier-resolution=[node]` flag can be used to customize
```

Comment:
I don't believe this change is correct. "mode" here is referring to the parameter name, whereas "node" is one possible value for the "mode" parameter. The "mode" could also have a value of "explicit".